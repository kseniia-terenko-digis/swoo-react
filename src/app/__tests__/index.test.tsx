import { shallow } from 'enzyme';
import App from '..';

describe('App component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<App />);

    expect(wrapper).toMatchSnapshot();
  });
});
