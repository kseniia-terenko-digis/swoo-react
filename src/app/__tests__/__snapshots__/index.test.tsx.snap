// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches snapshot 1`] = `
<Component
  theme={
    Object {
      "colors": Object {
        "black": "#000",
        "blue": "#004761",
        "green": "#8cd6bd",
        "white": "#fff",
      },
    }
  }
>
  <MainPage />
</Component>
`;
